pkg/splunk/common/util_test.go

```diff
@@ -572,12 +572,12 @@ func TestGetLabels(t *testing.T) {
 		}
 	}
 
-	test("indexer", "cluster-master", "t1", "t1", map[string]string{
+	test("indexer", CM, "t1", "t1", map[string]string{
 		"app.kubernetes.io/managed-by": "splunk-operator",
 		"app.kubernetes.io/component":  "indexer",
-		"app.kubernetes.io/name":       "cluster-master",
+		"app.kubernetes.io/name":       CM,
 		"app.kubernetes.io/part-of":    "splunk-t1-indexer",
-		"app.kubernetes.io/instance":   "splunk-t1-cluster-master",
+		"app.kubernetes.io/instance":   ManagerT1,
```

Comment:
Naming convention has been changed.. any reason why we just didn't use "cluster-manager" & "splunk-t1-cluster-manager" ?